package.json

```diff
@@ -58,7 +60,9 @@
     "uuid": "^9.0.0"
   },
   "devDependencies": {
+    "@playwright/test": "^1.40.0",
     "@rollup/plugin-commonjs": "^24.0.1",
+    "@types/node": "^20.9.2",
```

Comment:
This (and `@ts-check` in `playwright.config.js`) is probably not really needed right now, as TS won't run without a `tsconfig.json`.